modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-3x-major.adoc

```diff
@@ -184,6 +184,26 @@ In version 3.x of this connector, the polling mode of the Subscriber source is s
 </anypoint-mq:subscriber>
 ----
 
+This table lists the 2.x `polling` parameters and their 3.x equivalents.
+
+
+[%header,cols="20a,30a,20a,30a"]
+|===
+| Version 2.x Parameter |Description| Version 3.x Equivalent |Description
+|`fetchSize`| Declared at the `config` level, controls the size of the local message buffer, which is calculated as three times the `fetchSize` value.
+|`fetchSize`| Declared at the `Subscriber` level, controls the size of the local message buffer, which is calculated as three times the `fetchSize` value.
```

Comment:
Please change to:
"Sets the maximum amount of messages to be fetched on each polling execution, in range from 1 to 10 messages"

This is based on the Javadoc of the parameter: 
https://github.com/mulesoft/anypoint-mq-connector/blob/develop-4.x/anypoint-mq-extension/src/main/java/com/mulesoft/extension/mq/api/source/PollingTypeSubscriberFactory.java#L57